.github/workflows/default.yml

```diff
@@ -0,0 +1,45 @@
+name: Package Excalibur with Pyinstaller
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.config.os }}
+
+    strategy:
+      matrix:
+        config:
+          - { os: macos-latest, arch: x64, name: macos-latest }
+          - { os: ubuntu-latest, arch: x64, name: ubuntu-latest }
+          - { os: windows-latest, arch: x64, name: windows-latest }
+          - { os: windows-latest, arch: ia32, name: windows-latest-32 }
```

Comment:
@arky Will `windows-latest` build it for windows 10? Is that build going to work on windows 7 which #99 is about? I don't have a windows 7 / 10 to test out the build.

Reply:
But I guess we can run this workflow and build it anyway.

Reply:
@vinayak-mehta According to Github Actions documentation the default windows-latest OS is Windows Server 2019 https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners

The workflow with generate an artifact with '-ia32' suffix that could be tested on Windows 32bit environments. That will help us resolve the #99.